covid19-etl/etl/cityofchicago.py

```diff
@@ -0,0 +1,410 @@
+            "age_70_79": "70 to 79",
+            "age_80_": "greater than 80",
+            "80_": "greater than 80",
+            "age_unknown": "unknown",
+            "unknown_age": "unknown",
+        }
+
+        # Race mapping for submitter id value from original dataset to value in Gen3 data dictionary
+        race_submitter_id = {
+            "latinx": "Hispanic or Latino",
+            "asian_non_latinx": "Asian",
+            "black_non_latinx": "Black or African-American",
+            "white_non_latinx": "White",
+            "other_non_latinx": "Other race",
+            "other_race_non_latinx": "Other race",
+            "unknown_race_eth": "Unknown",
+            "unknown_race_ethnicity": "Unknown",
+        }
+
+        # race mapping for value from original dataset to value in Gen3 data dictionary
+        race = {
+            "latinx": "Hispanic",
+            "asian_non_latinx": "Asian",
+            "black_non_latinx": "Black",
+            "white_non_latinx": "White",
+            "other_non_latinx": "Other",
+            "other_race_non_latinx": "Other",
+            "unknown_race_eth": "Unknown",
+            "unknown_race_ethnicity": "Unknown",
+        }
+
+        # Gender for value from original dataset to value in Gen3 data dictionary
+        gender = {
+            "female": "Female",
+            "male": "Male",
+            "unknown_gender": "Unknown",
+        }
+
+        record_type_mapping = {
+            "cases": "count",
+            "deaths": "deaths",
+            "hospitalizations": "hospitalizations",
+        }
+
+        record_type, submitter_value = header_value.split("_", maxsplit=1)
+        if submitter_value in age_group:
+            if "80" in submitter_value:
+                age_group_dict["AgeGroup"] = "80+"
+            else:
+                age_group_dict["AgeGroup"] = age_group[submitter_value].replace(
+                    " to ", "_"
+                )
+            summary_group_demographics_value_dict["age_group"] = age_group[
+                submitter_value
+            ]
+
+        elif submitter_value in race:
+            age_group_dict["Race"] = race_submitter_id[submitter_value]
+            summary_group_demographics_value_dict["race"] = race[submitter_value]
+            if submitter_value == "latinx":
+                summary_group_demographics_value_dict["ethnicity"] = "Hispanic"
+            else:
+                summary_group_demographics_value_dict["ethnicity"] = "Nonhispanic"
+
+        elif submitter_value in gender:
+            age_group_dict["Gender"] = gender[submitter_value]
+            if submitter_value == "unknown_gender":
+                summary_group_demographics_value_dict[
+                    "gender"
+                ] = "Unknown or Left Blank"
+            else:
+                summary_group_demographics_value_dict["gender"] = gender[
+                    submitter_value
+                ]
+
+        else:
+            return
```

Comment:
is it expected that we'd get a header that doesn't match any of the previous `if` statements? if not, we should raise an error here instead of silently returning

Reply:
I am adding code to raise Exception here.


```
        else:
            raise Exception("This process only give summary of covid data on either AgeGroup, Race or Gender.")
```